e2e/wdio/driver/wdio.ts

```diff
@@ -1,226 +1,271 @@
-export class Wdio {
+export function defaultWaitTime(): number {
+    return browser.options.waitforTimeout;
+}
 
-    get defaultWaitTime(): number {
-        return browser.options.waitforTimeout;
-    }
+export function currentPlatformName(): string {
+    return browser.capabilities.platformName;
+}
 
-    get currentPlatformName(): string {
-        return browser.capabilities.platformName;
-    }
+export function open(path: string = ''): void {
+    browser.url(path);
+}
 
-    open(path: string = ''): void {
-        browser.url(path);
-    }
+export function pause(waitTime: number = defaultWaitTime()): void {
+    browser.pause(waitTime);
+}
 
-    pause(waitTime: number = this.defaultWaitTime): void {
-        browser.pause(waitTime);
-    }
+export function isBrowser(browserName: string): boolean {
+    return browser.capabilities.browserName === browserName;
+}
 
-    isBrowser(browserName: string): boolean {
-      return browser.capabilities.browserName === browserName;
+export function browserIsIEorSafari(): boolean {
+    if (browserIsSafari()) {
+        return true;
+    } else if (browserIsIE) {
```

Comment:
this function doesn't work. please recheck it. 
fixed in my new branch with 
if ( this.isBrowser('Safari') { 
return true
} else if (this.isBrowser('internet explorer) {
return true
}
return false

Reply:
probably could do it with if ( browserIsSafari() === true) ...